### ISSUE TITLE
Link to billing account cloud console

### ISSUE THREAD

**Issue by bradtaylor** [DataBiosphere/saturn-ui#1193]
As a program manager with a billing account, I want to see how much money has been spent this month on various cloud resources.

Eventually Terra may take on more of these features directly. For now, we will provide a link out to the cloud console for this billing account, where users can access this and other information

**Comment by katevoss** [comment]
To clarify, this is a link to the billing **account**, not the billing **project**, correct?

**Comment by katevoss** [comment]
Format will be `https://console.cloud.google.com/billing/<billingAccountId>/budgets?authuser=?authuser=${Auth.getUser().email}`

**Comment by breilly2** [comment]
Correct. "Billing Project" is a FireCloud concept that kinda sorta combines the Google Cloud concepts of "Billing Account" and "Project".

**Comment by katevoss** [comment]
This issue is fixed by a commit on another branch: https://github.com/DataBiosphere/terra-ui/commit/e3ce6cd2cb85e0759bddd21d8cc7d6bcbbfe508a which is for the issue #1188

**Comment by katevoss** [comment]
This issue is now deprioritized and going to the backlog.
